gum/backend-windows/gummemoryaccessmonitor-windows.c

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2010, 2015 Ole André Vadla Ravnås <ole.andre.ravnas@tillitech.com>
+ * Copyright (C) 2015 Eloi Vanderbeken <eloi.vanderbeken@synacktiv.com> - added the possibility to monitor operations by mask and to auto-reset or not the page monitoring after the first hit.
+ * Copyright (C) 2010, 2015 Ole André Vadla Ravnås <ole.andre.ravnas@tillitech.com> - original code
```

Comment:
I don't think we should put change-logs details here, we already have git history for that.